<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JPTMainWindow</class>
 <widget class="QMainWindow" name="JPTMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JPTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="messagesTextBrowser">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>451</width>
      <height>79</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>631</width>
      <height>311</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="sendTab">
     <attribute name="title">
      <string>Outbox</string>
     </attribute>
     <widget class="QLabel" name="outboxLabel">
      <property name="geometry">
       <rect>
        <x>294</x>
        <y>13</y>
        <width>52</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Outbox:</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="outboxTextBrowser">
      <property name="geometry">
       <rect>
        <x>294</x>
        <y>37</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="packetOutboxLabel">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>13</y>
        <width>92</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Packet Outbox</string>
      </property>
     </widget>
     <widget class="QListWidget" name="packetOutboxListWidget">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>37</y>
        <width>200</width>
        <height>192</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>175</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="clearOutboxButton">
      <property name="geometry">
       <rect>
        <x>294</x>
        <y>239</y>
        <width>75</width>
        <height>32</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="receiveTab">
     <attribute name="title">
      <string>Inbox</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QFrame" name="p2Frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="p2packetInboxLabel">
              <property name="text">
               <string>P2 Packet Inbox:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="p2packetInboxListWidget">
              <property name="maximumSize">
               <size>
                <width>130</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="p2InboxLabel">
              <property name="text">
               <string>P2 Inbox:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="p2Inbox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="p3Frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="p3packetInboxLabel">
              <property name="text">
               <string>P3 Packet Inbox:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="p3packetInboxListWidget">
              <property name="maximumSize">
               <size>
                <width>130</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="p3InboxLabel">
              <property name="text">
               <string>P3 Inbox:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="p3Inbox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Garbage:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTextBrowser" name="garbageInbox">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="byteStagingLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="leftTitleLabel">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>13</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>JPTest Configuration</string>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>45</y>
      <width>200</width>
      <height>651</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="portSettingsPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>602</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Configure Test</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>JAGUAR Role</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>My JAGUAR ID:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="JAGIDcomboBox">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="jptestServerRButton">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>JPTest Server</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="jptestClientRButton">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>JPTest Client</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Test Mode Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QCheckBox" name="stepModeCheckBox">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Step mode</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="noMSP430checkBox">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&quot;No MSP430&quot; mode</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Working Directory</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="workingDirLineEdit">
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="browseWkDirButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Port Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="refreshPortsButton">
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Select Port:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QListWidget" name="portListWidget">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="testSettingsPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>513</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Select JPTest</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Select Test to run:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="jptestListWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="msgsLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Messages:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearMessagesButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>520</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <zorder>leftTitleLabel</zorder>
   <zorder>toolBox</zorder>
   <zorder>messagesTextBrowser</zorder>
   <zorder>tabWidget</zorder>
   <zorder>msgsLabel</zorder>
   <zorder>clearMessagesButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
